Makefile

```diff
@@ -38,6 +38,16 @@ compile-darwin: deps-only
 compile-windows: deps-only
 	@echo "=== $(PROJECT_NAME) === [ compile-windows    ]: building commands:"
 	@GOOS=windows $(GO_CMD) build -o $(BIN_DIR)/$(BINARY_NAME).exe ./cmd/...
+tools-vcsim-run:
```

Comment:
wdyt of adding the `compile-linux` target as a dependency of this and mount all needed things as volumes in the docker-compose using directly the infrastructure image, without having to build a custom image ?

Reply:
done